config/v3_2_experimental/config.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 Red Hat, Inc.
```

Comment:
Most of these files have different / non-current years.

Reply:
Dates updated.